apps/backend/src/modules/users/libs/types/types.ts

```diff
@@ -1,4 +1,5 @@
 export {
+	type PaginationParameters,
```

Comment:
I think it's better to move importing `PaginationParameters` to common types. It's not related only to user model and will be used in others as well